auth/src/main/java/com/hms/lib/commonmobileservices/auth/AuthService.kt

```diff
@@ -48,17 +57,28 @@ interface AuthService {
     fun updateUsername(username: String?): Work<Unit>
     fun updateEmail(email: String?, verifyCode: String?): Work<Unit>
     fun updatePhone(countryCode: String?, phoneNumber: String?, verifyCode: String?): Work<Unit>
-    fun updatePasswordwEmail(password: String?, verifyCode: String?): Work<Unit>
-    fun updatePasswordwPhone(password: String?, verifyCode: String?): Work<Unit>
+    fun updatePasswordWithEmail(password: String?, verifyCode: String?): Work<Unit>
```

Comment:
These methods don't get email and phone. So, we can change method name as UpdatePassword

Reply:
resolved.